aspnetcore/signalr/dotnet-client.md

```diff
@@ -34,7 +32,24 @@ Install-Package Microsoft.AspNetCore.SignalR.Client
 
 To establish a connection, create a `HubConnectionBuilder` and call `Build`. The hub URL, protocol, transport type, log level, headers, and other options can be configured while building a connection. Configure any required options by inserting any of the `HubConnectionBuilder` methods into `Build`. Start the connection with `StartAsync`.
 
-[!code-csharp[Build hub connection](dotnet-client/sample/signalrchatclient/MainWindow.xaml.cs?name=snippet_MainWindowClass&highlight=14-16,32)]
+[!code-csharp[Build hub connection](dotnet-client/sample/signalrchatclient/MainWindow.xaml.cs?name=snippet_MainWindowClass&highlight=15-17,42)]
+
+### Handle lost connection
+
+You can use the `Closed` event to respond to a lost connection. For example, you might want to automate reconnection.
```

Comment:
~~You can use the~~
Use the

Link to the API ref page for the `Closed` event. You can use the following syntax:

```
<xref:Microsoft.AspNetCore.SignalR.Client.HubConnection.Closed>
```